public/index.php

```diff
@@ -32,12 +37,52 @@ function redirect($url)
         } elseif ($_SERVER['REQUEST_METHOD']==='POST') {
             $username = $_POST['username'];
             $password = $_POST['password'];
-            echo sprintf("%s %s", $username, $password);
+            $reader = new ReaderController();
+            if ($reader->login($username, $password) == true) {
+                #if (session_status() == PHP_SESSION_NONE) {
+                    # initially the session is null, if it has a value then somebody is logged in
+                if (isset($_SESSION['user'])) {
+                    echo "Somebody else was logged in";
+                }
+                $_SESSION['user'] = $username;
+                var_dump($_SESSION['user']);
+                redirect('/home');
+            }
+        }
+        break;
+
+    case '/signUp':
+        if ($_SERVER['REQUEST_METHOD']==='GET') {
+            require 'html/signUp.html';
+        } elseif ($_SERVER['REQUEST_METHOD']==='POST') {
+            $username = $_POST["username"];
+            $password = $_POST["password"];
+            $password2 = $_POST['password2'];
+            if ($password != $password2) {
```

Comment:
Never use numbers in variable names.
Try to avoid non-strict comparisons.
All the logic regarding the data processing should be moved to the Controller, Services, Repository, etc.